### PR TITLE
feature: add new setting items with i18n content

### PR DESCRIPTION
**Pull request by bzp2010**
add new setting items with i18n content for AdminAPIConfig

### CODE REVIEW

File: src/pages/Settings.tsx [hunk 1]
```diff
@@ -1,14 +1,16 @@
 import React, {useEffect} from "react";
 import {useForm} from "antd/es/form/util";
-import {Button, Card, Form, Input, notification} from "antd";
+import {Button, Card, Form, Input, notification, Select} from "antd";
 import {formatMessage, FormattedMessage} from "umi-plugin-react/locale";
 import {router} from "umi";
 import {PageHeaderWrapper} from "@ant-design/pro-layout";
-import {getAdminAPI, getAdminAPIKey} from "@/utils/utils";
+import {getAdminAPIConfig} from "@/utils/utils";
```
Comment (juzhiyuan) [inline src/pages/Settings.tsx:7]: It's not a good idea to name as `/utils/utils`, please use something like `/utils/setting` or `/utils/config` instead.

Comment (juzhiyuan) [reply src/pages/Settings.tsx:7]: or use `/utils` file directly.

File: src/pages/Settings.tsx [hunk 2]
```diff
@@ -17,22 +19,27 @@ const layout = {
...
-    offset: 2,
+    offset: 4,
   },
 };
 
 const Settings: React.FC = () => {
   const [form] = useForm();
 
   useEffect(() => {
+    let adminAPIConfig = getAdminAPIConfig();
```
Comment (juzhiyuan) [inline src/pages/Settings.tsx:30]: please use ES6 destructing here https://es6.ruanyifeng.com/#docs/destructuring

```js
const {adminAPISchema, adminAPIHost, ....} = getAdminAPIConfig()
```

Comment (juzhiyuan) [reply src/pages/Settings.tsx:30]: maybe use `schema, host, path, key` instead of `adminAPIxxx`?

File: src/pages/Settings.tsx [hunk 3]
```diff
@@ -47,10 +54,33 @@ const Settings: React.FC = () => {
       <Card>
         <Form {...layout} form={form} onFinish={onFinish}>
           <Form.Item
-            label="Admin API"
-            name="adminAPI"
+            label="Admin API Schema"
+            name="adminAPISchema"
```
Comment (juzhiyuan) [inline src/pages/Settings.tsx:58]: i18n

File: src/locales/zh-CN/settings.ts [hunk 4]
```diff
@@ -1,3 +1,5 @@
 export default {
-  'app.settings.description.invalid-admin-api': '非法的管理API'
+  'app.settings.description.invalid-admin-api-schema': '非法的管理API协议',
```
[the remark this answers is not in the document]
Comment (bzp2010) [reply src/locales/zh-CN/settings.ts:2]: Ok

File: src/pages/Settings.tsx (hunk 2 above)
Comment (bzp2010) [reply src/pages/Settings.tsx:30]: Although we should use the ES6 standard, this may cause the host or path to conflict with other content in the future.

Comment (bzp2010) [reply src/pages/Settings.tsx:30]: `schema, host, path, key` will be changed to

File: src/pages/Settings.tsx (hunk 3 above)
Comment (bzp2010) [reply src/pages/Settings.tsx:58]: Ok

File: src/pages/Settings.tsx (hunk 1 above)
Comment (bzp2010) [reply src/pages/Settings.tsx:7]: use `/utils/setting`

**Comment by juzhiyuan** [comment]
@bzp2010 Is this PR ready to merge?

**Comment by bzp2010** [comment]
Yes, it's ready!

**Pull request merged**